aws_lambda_powertools/tracing/tracer.py

```diff
@@ -15,7 +15,8 @@
 )
 from aws_lambda_powertools.shared.lazy_import import LazyLoader
 from aws_lambda_powertools.shared.types import AnyCallableT
-from aws_lambda_powertools.tracing.base import BaseProvider, BaseSegment
+from aws_lambda_powertools.tracing.provider import XrayProvider
```

Comment:
Again, I understand that you are removing the other providers; however, the manner in which you have written the code results in importing all providers despite utilizing only one of them.